bin/dfx-sns-demo-mksns-parallel

```diff
@@ -70,7 +71,7 @@ become_user() {
 create_all_sns() {
   PROCESSES=()
   echo "I am $$"
-  for ((i = 1; i <= NUM_SNS_TO_MAKE; i++)); do
+  for ((i = INDEX_OFFSET; i <= NUM_SNS_TO_MAKE; i++)); do
```

Comment:
Can we apply the offset only on line 81 and leave line 77 zero based as before?

Reply:
What I mean is that the entire for loop should stay unchanged and we just apply the offset when passing the index. Hence the "index offset".

Also, I think currently it actually doesn't work because you don't apply the offset to the range end. You just implicitly add 1 to it by changing `<` to `<=`.